### ISSUE TITLE
Backport menu from 4.X to choose instruments/sounds in mixer

### ISSUE THREAD

**Issue by zorba77** [Jojo-Schmitz/MuseScore#699]
In mscore 3.x's mixer instruments are not grouped for soundfont, so it's very difficult to navigate the instruments (and understand the origin of the sound)

![image](https://github.com/user-attachments/assets/1d761ca5-4b06-46ff-8a36-b752a03bcf5b)

Mscore 4.X has a better system:

![image](https://github.com/user-attachments/assets/8bb93d34-0804-42c2-b3da-6fea6538046b)

Is it possible to backport this menu? It's very useful

**Comment by mkj42** [comment]
If I may add my input:
More sorted and more organized would be great, 
but please do not in any way change how instruments are chosen and assigned in 3.7.
The default 'choose automatically' in 4 gives you no idea which sound is actually in use.
Also, please do not replace the soundfont with MS Basic.sf3; it still sounds awful by comparison.
https://github.com/musescore/MuseScore/issues/23659#issuecomment-2253113567

**Comment by Jojo-Schmitz** [comment]
I have no plans whatsoever to replace any soundfont, and esp. not with MS Basic.sf3

**Comment by zorba77** [comment]
I agree, but if you need to add more soundfonts,  like Sonatina, the actual sistem is very confusing.
All instruments are listed together, the 4.X listing is clearer.
Is it possible to backport that sistem?

**Comment by Jojo-Schmitz** [comment]
I doubt it, it'd be a full reimplementation

**Comment by zorba77** [comment]
Only the menu to choose instruments, not the entire sound system

**Comment by zorba77** [comment]
And append in the list the name of the soundfont? Now it's hard to understand the origin's soundfont of an instrument 
This is easier

**Comment by zorba77** [comment]
Any system that help to understand which is the origin' soundfont.
Or, if the 4.X way is require too time to implement, a menu like this is doable?
Soundfont->instrument
So, when I click to choose the instrument I can see:

Sonatina
    |-------- list with various instruments
Internal default
    |-------- list with various instruments
 Other soundfont
    |-------- list with various instruments
